.drone.star

```diff
@@ -70,22 +78,20 @@ config = {
         'webUISharingPermissionsUsers': 'webUISharingPermissionsUsers',
         'webUISharingFilePermissionsGroups': 'webUISharingFilePermissionsGroups',
         'webUISharingFolderPermissionsGroups': 'webUISharingFolderPermissionsGroups',
-        'webUISharingFolderAdvancedPermissionsGroups': 'webUISharingFolderAdvPermissionsGrp',
+        'webUISharingFolderAdvPermissionsGrp': 'webUISharingFolderAdvancedPermissionsGroups',
```

Comment:
Note: The key is the "short name" for the drone pipeline, the value (right side) is the actual name of the test suite to run.